src/Common/tests/Tests/System/IO/PathInternal.Tests.cs

```diff
@@ -1,91 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.IO;
-using System.Text;
-using Xunit;
-
-namespace Tests.System.IO
-{
-    public class PathInternalTests
-    {
-        [Theory,
-            InlineData("Foo", "Foos"),
-            InlineData("Foo", null)]
-        public void StartsWithOrdinal_NegativeCases(string source, string value)
```

Comment:
I'll call it out in #27155. It can't be in S.R.E as we don't want to add InternalsVisbileTo. The strategy I was attempting was just including the source files in the Common test project- we can continue to do so or perhaps look at creating a similar project in CoreCLR.